core/src/test/scala/org/broadinstitute/dsp/workbench/hamm/TestExecutionContext.scala

```diff
@@ -0,0 +1,20 @@
+package org.broadinstitute.dsp.workbench.hamm
+
+import java.util.concurrent.{ExecutorService, Executors}
+
+import scala.concurrent.ExecutionContextExecutor
+
+object TestExecutionContext {
+  implicit val testExecutionContext = new TestExecutionContext()
+}
+
+class TestExecutionContext() extends ExecutionContextExecutor {
```

Comment:
why does this need to be a separate class?

Reply:
I was told it was good practice - this allows people to instantiate a separate TestExecutionContext if they want